rpm/dbt2-pgsql-plpgsql.spec

```diff
@@ -0,0 +1,41 @@
+%global debug_package %{nil}
+%{!?pkgrevision: %global pkgrevision 1}
+%{!?pgversion: %global pgversion 14}
+%define installpath /usr/share
+%global pkgname dbt2-pgsql-plpgsql_%{pgversion}
+%define _unpackaged_files_terminate_build 0
+
+Name:          %{pkgname}
+Version:       %{pkgversion}
+Release:       %{pkgrevision}%{?dist}
+Summary:       Fair Use TPC-C benchmark kit - PostgreSQL PL/pgsql Stored Functions
+License:       The Artistic License
+Source:        v%{version}.zip
+BuildRoot:     %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+Requires:      postgresql%{pgversion}-server, llvm-libs, postgresql%{pgversion}
```

Comment:
I don't think we should depend on `postgresql` packages because in some cases (cloud context for example), this package will be installed on the "client" side only. Technically, it doesn't require having Postgres server binaries installed on the system.
In the same way, I don't think we should build one specific package for each Postgres version, that's just SQL files that can be loaded on all Postgres versions and flavors.

I'd suggest naming this package `dbt2-pgsql-plpgsql.rpm` and not making it dependent on any other package.

Reply:
Agreed.  All oversights on my part.  Push updates.